pkg/identityserver/store/gateway_antenna.go

```diff
@@ -29,6 +29,8 @@ type GatewayAntenna struct {
 	Gain float32
 
 	Location
+
+	Placement int
```

Comment:
We prefer to not do any database migrations in patches, so unless this is critical I would rather see this go to 3.14.

Reply:
Please see target branch

Reply:
Milestone was wrong, fixed that